draft-ietf-scitt-architecture.md

```diff
@@ -609,7 +615,7 @@ This verifiable data structure MUST support the following security requirements:
 
 Append-Only:
 
-: once included in the verifiable data structure, a Signed Statement cannot be modified, deleted, or reordered; hence its Receipt provides an offline verifiable proof of Registration.
+: once committed to the verifiable data structure, a Signed Statement cannot be modified, deleted, or reordered; hence its Receipt provides an offline verifiable proof of Registration.
```

Comment:
```suggestion
: A property required for a verifiable data structure to be applicable to SCITT, ensuring that the Statement Sequence cannot modified, deleted or reordered.
```

Reply:
consider moving this definition above "Append-Only Log"

Reply:
Note: I've tweaked a bit to add "...be modified..."